microsoft/testsuites/security/fips.py

```diff
@@ -32,61 +137,84 @@ class Fips(TestSuite):
         4. Verify that FIPS was enabled properly
         """,
         priority=3,
+        requirement=simple_requirement(
+            unsupported_os=[CBLMariner],
```

Comment:
Why azl is unsupported?

Reply:
if it's by design, I prefer to have the same `verify_fips_enable` for all distros. Azl can go with a separated path.